ui/narrative/methods/pickaxe/spec.json

```diff
@@ -42,6 +42,12 @@
                         "display": "Spontaneous Rules",
                         "id": "spont",
                         "ui_name": "Spontaneous Rules"
+                    },
+                    {
+                        "value": "retro_rules_2",
+                        "display": "Retro Rules 2",
```

Comment:
Maybe "RetroRules Diameter 2"?

Reply:
Yes, thinking along the same lines, just want to test with the smaller file first, then will add the names that makes more sense.